api/apps/api/src/modules/clone/export/adapters/export-adapters.module.ts

```diff
@@ -6,15 +6,25 @@ import { ExportRepository } from '../application/export-repository.port';
 import { ResourcePieces } from '../application/resource-pieces.port';
 import { ArchiveCreator } from '../application/archive-creator.port';
 import { ResourcePiecesAdapter } from './resource-pieces.adapter';
-import { InMemoryExportRepo } from './in-memory-export.repository';
 import { NodeArchiveCreator } from './node-archive-creator';
+import { TypeormExportRepository } from '@marxan-api/modules/clone/export/adapters/typeorm-export.repository';
+import { ExportEntity } from '@marxan-api/modules/clone/export/adapters/entities/exports.api.entity';
+import { ExportComponentEntity } from '@marxan-api/modules/clone/export/adapters/entities/export-components.api.entity';
+import { ComponentLocationEntity } from '@marxan-api/modules/clone/export/adapters/entities/component-locations.api.entity';
```

Comment:
relative imports (:

Reply:
Do we have a documented rule of when to use absolute/relative imports? I've seen multiple comments on PRs around this, so I'm wondering if everyone is aware of exactly how we should approach this - I am not, for one.

Reply:
we don't, and I got lost in the evolution of this through time.

My preference would be:
- within a Nest module, relative paths
- across modules within the same service, "root-relative" paths
- across services or from libs, `@<whatever>` "root-relative" paths

though IDEs should make the difference between the first two cases mostly cosmetic - personally I like "root-relative" paths as I find that it removes the small cognitive overload that (at least for me) relative paths introduce by having to remember or figure out "where am I" in the paths hierarchy to then travel up and down the tree mentally. But even this, I normally deal with ok by just ctrl-clicking through the relevant imports if I can't figure them out at a glance.